stdlib/http/src/main/ballerina/src/http/resiliency/load_balance_client_endpoint.bal

```diff
@@ -304,37 +304,27 @@ function populateGenericLoadBalanceActionError(LoadBalanceActionErrorData loadBa
 
 # The configurations related to the load balance client endpoint.
 #
-# + httpVersion - The HTTP version to be used to communicate with the endpoint
-# + http1Settings - Configurations related to HTTP/1.x protocol
-# + http2Settings - Configurations related to HTTP/2 protocol
-# + timeoutInMillis - The maximum time to wait (in milli seconds) for a response before closing the connection
-# + forwarded - The choice of setting forwarded/x-forwarded header
-# + followRedirects - Redirect related options
-# + poolConfig - Configurations associated with request pooling
+# httpVersion - Copied from CommonClientConfiguration
+# http1Settings - Copied from CommonClientConfiguration
+# http2Settings - Copied from CommonClientConfiguration
+# timeoutInMillis - Copied from CommonClientConfiguration
+# forwarded - Copied from CommonClientConfiguration
+# followRedirects - Copied from CommonClientConfiguration
+# poolConfig - Copied from CommonClientConfiguration
+# cache - Copied from CommonClientConfiguration
+# compression - Copied from CommonClientConfiguration
+# auth - Copied from CommonClientConfiguration
+# circuitBreaker - Copied from CommonClientConfiguration
+# retryConfig - Copied from CommonClientConfiguration
+# cookieConfig - Copied from CommonClientConfiguration
 # + proxy - Proxy related options
 # + targets - The upstream HTTP endpoints among which the incoming HTTP traffic load should be distributed
-# + cache - The configurations for controlling the caching behaviour
-# + compression - Specifies the way of handling compression (`accept-encoding`) header
-# + auth - Configurations related to the HTTP authentication.
-# + circuitBreaker - Circuit Breaker configuration
-# + retryConfig - Retry related options
 # + lbRule - LoadBalancing rule
 # + failover - Configuration for load balancer whether to fail over in case of a failure
 public type LoadBalanceClientConfiguration record {|
-    string httpVersion = HTTP_1_1;
-    ClientHttp1Settings http1Settings = {};
-    ClientHttp2Settings http2Settings = {};
-    int timeoutInMillis = 60000;
-    string forwarded = "disable";
-    FollowRedirects? followRedirects = ();
+    *CommonClientConfiguration;
     ProxyConfig? proxy = ();
```

Comment:
We can remove ProxyConfig as well as it is already grouped under ClientHttp1Settings type

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/21275/commits/2bbf173f2e61c966f125812b80ddcdaf81b4ccff